src/test/llmnode_test.cpp

```diff
@@ -312,11 +312,11 @@ TEST_F(LLMFlowHttpTest, inferChatCompletionsUnary) {
         handler->dispatchToProcessor(endpointChatCompletions, requestBody, &response, comp, responseComponents, &writer),
         ovms::StatusCode::OK);
     // Assertion split in two parts to avoid timestamp missmatch
-    const size_t timestampLength = 10;
+    // const size_t timestampLength = 10;
     std::string expectedResponsePart1 = R"({"choices":[{"finish_reason":"stop","index":0,"logprobs":null,"message":{"content":"\nOpenVINO is","role":"assistant"}}],"created":)";
     std::string expectedResponsePart2 = R"(,"model":"llmDummyKFS","object":"chat.completion"})";
-    ASSERT_EQ(response.compare(0, expectedResponsePart1.length(), expectedResponsePart1), 0);
-    ASSERT_EQ(response.compare(expectedResponsePart1.length() + timestampLength, expectedResponsePart2.length(), expectedResponsePart2), 0);
+    // TODO: New output ASSERT_EQ(response.compare(0, expectedResponsePart1.length(), expectedResponsePart1), 0);
+    // TODO: New output ASSERT_EQ(response.compare(expectedResponsePart1.length() + timestampLength, expectedResponsePart2.length(), expectedResponsePart2), 0);
```

Comment:
What's the point of the test if we do not assert the result now?

Reply:
The checks in such form are prone to runtime and tokenizer version changes. Currently they are also HW dependent and cause more issues than give benefits. We should think of more robust accuracy checks later on.

Reply:
We have entire set of tests added lately that base on tokenzier output: https://github.com/openvinotoolkit/model_server/commit/2e7fc7744f553bf5f71c27416a9667d2b94dcd80
(And probably more new ones that I might not know of) @mzegla

Reply:
I'm aware, but they do not give us confidence we need. Now @rasapala switched OV and tokenizers version and all those assertions started to fail. Can we afford updating such amount of checks in worst case scenario with every OV/tokenizers update? + the fact that currently those assertion pass on one machine does not guarantee they pass on another

Reply:
I completely understand and agree.
I'm just saying that without those checks, the tests are useless, we could also remove them entirely, so they do not give us false confidence that we actually test something

Reply:
It is ok to comment those assertions for now but we should in some extent assert the response. It is not meant to be used for accuracy testing but at least to confirm if the response is a string with non zero length. We need to update those assertions shortly.